src/internal/Notification.ts

```diff
@@ -245,20 +218,11 @@ export class Notification<T> {
  * @param observer The observer to notify.
  */
 export function observeNotification<T>(notification: ObservableNotification<T>, observer: PartialObserver<T>) {
-  if (typeof notification.kind !== 'string') {
+  const { kind, value, error } = notification as any;
+  if (typeof kind !== 'string') {
     throw new TypeError('Invalid notification, missing "kind"');
   }
-  switch (notification.kind) {
-    case 'N':
-      observer.next?.(notification.value!);
-      break;
-    case 'E':
-      observer.error?.(notification.error);
-      break;
-    case 'C':
-      observer.complete?.();
-      break;
-  }
+  kind === 'N' ? observer.next?.(value!) : kind === 'E' ? observer.error?.(error) : observer.complete?.();
```

Comment:
If kept as ternaries this needs newline formatting otherwise it's _really_ tough to understand. Has the team considered using prettier maybe?

Reply:
I tend to agree. We're using prettier, however, I think maybe the line length is too long.

Reply:
I think we'll probably just change the line length to the default in another PR. That should correct this issue.